lib/NetPacks.h

```diff
@@ -69,15 +69,14 @@ struct StackLocation
 /***********************************************************************************************************/
 
 
-struct PackageApplied : public CPackForClient
+struct DLL_LINKAGE PackageApplied : public CPackForClient
 {
 	PackageApplied()
 		: result(0), packType(0),requestID(0)
 	{}
 	PackageApplied(ui8 Result)
 		: result(Result), packType(0), requestID(0)
-	{}
-	void applyCl(CClient *cl);
+	{}
	virtual void visitTyped(ICPackVisitor & visitor) override;

```

Comment:
Weird formatting? Should be end of line instead of... whatever is "^M" is
(not just here, search shows 18 hits)

Reply:
@IvanSavenko it is windows EOL, should not be used in UNIX.

Reply:
Maybe something. A lot of changes were done using some regex replaces) It will be tested and cleaned up. Just first building version